package.json

```diff
@@ -4,31 +4,38 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
-    "eslint": "eslint --ignore-path .gitignore .",
     "dev": "export NODE_ENV=development && nodemon src/temp-server.js ",
-    "test": "export NODE_ENV=testing && nodemon src/temp-server.js "
+    "test": "export NODE_ENV=testing && nodemon src/temp-server.js ",
+    "dev-start": "nodemon --exec babel-node ./src/index.js",
```

Comment:
@victorkarangwa4 no need to have both `dev-start` and `dev ` scripts. lets use one